src/Extensions/Diagnostics.HealthChecks/IHealthStatusSender.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+
+namespace Microsoft.Omex.Extensions.Diagnostics.HealthChecks
+{
+	/// <summary>
+	/// Interface for sending health status from HealthPublisher to extenal consumer (ex. SF Cluster).
+	/// </summary>
+	public interface IHealthStatusSender
+	{
+		/// <summary>
+		/// Initialization method that's called before each health status publishing session.
+		/// </summary>
+		Task IntializeAsync(CancellationToken token);
```

Comment:
I take it the token is just added to enforce standard convention and avoid potential future refactoring?
*It's not used in the concrete implementation below.

Reply:
Good point, I've found a way to propagate `CancelationToken` into client creation.